setup.py

```diff
@@ -0,0 +1,58 @@
+"""A setuptools based setup module.
+
+See:
+https://packaging.python.org/guides/distributing-packages-using-setuptools/
+https://github.com/pypa/sampleproject
+https://github.com/pypa/sampleproject/blob/master/setup.py
+"""
+
+# Always prefer setuptools over distutils
+from setuptools import setup, find_packages
+from os import path
+# io.open is needed for projects that support Python 2.7
+# It ensures open() defaults to text mode with universal newlines,
+# and accepts an argument to specify the text encoding
+# Python 3 only projects can skip this import
+from io import open
```

Comment:
We can skip this, as we don't support Python 2.7.

Reply:
I missed it. Removed